app/src/layouts/kanban/kanban.vue

```diff
@@ -134,6 +144,7 @@ const props = withDefaults(
 		sortField?: string | null;
 		userField?: string | null;
 		groupsSortField?: string | null;
+		layoutOptions: LayoutOptions;
```

Comment:
`layoutOptions` were not "caught" as a prop in the Kanban layout, but it is already passed to the layout. Technically it might be possible to access the data through `$attrs` instead, but since I use the data, I figured this was best.